dotcom-rendering/src/components/marketing/epics/ThreeTierChoiceCards.tsx

```diff
@@ -100,20 +91,6 @@ const recommendedPillStyles = css`
 	right: ${space[5]}px;
 `;
 
-const discountedPillStyles = css`
```

Comment:
discount pill colours will come from the server when we get round to implementing proper support for discounts